src/e2e/specs/auth.spec.ts

```diff
@@ -27,7 +27,11 @@ test.describe(`${process.env.E2E_TEST_ENV} - Authentication flow verification @s
     await authPage.signUp(randomEmail, page);
 
     // assert successful login
-    expect(page.url()).toBe(`${process.env.E2E_TEST_BASE_URL}/user/dashboard`);
+    const successUrl =
+      process.env.E2E_TEST_ENV === "local"
```

Comment:
Why is it redirect different? Is it a recent PR that just hasn't landed on Stage/Prod yet?

Reply:
nope, there's some flags we don't have in stage that we have in local(manual managed by the team currently)